torch/_refs/__init__.py

```diff
@@ -413,6 +413,17 @@ def _ref(a: TensorLikeType) -> TensorLikeType:
     return inner
 
 
+def _make_alias(fn, name=None):
```

Comment:
Is `name=None` actually an interesting case to support? What's an example where a developer would want to do that instead of explicitly specifying a name?

Reply:
The parsing was wrong, and you were right. I think it's best to remove the parsing altogether.